core/domain/suggestion_services.py

```diff
@@ -147,7 +148,7 @@ def get_all_stale_suggestions():
 
 
 def _update_suggestion(suggestion):
-    """Updates the given suggestion.
+    """Updates the given suggestion in database.
```

Comment:
Do we really need this change? Could you explain why you made this change? Also if we do make this change, then 
```suggestion
    """Updates the given suggestion in the datastore.
```

Reply:
ok sir